### DISCUSSION TITLE
Does the Accordion (Disclosure / DisclosureGroups) support navigation through items using the up and down arrow keys?

### DISCUSSION THREAD

**Discussion by AndresMangano** [adobe/react-spectrum#7431] [General]
I'm not able to navigate through the accordion items using the keys, is this feature implemented? If it's not, as far as I understand for accessibility purposes Accordions have to be navigable using the up and down arrow keys.

**Comment by yihuiliao** [comment]
No, I don't believe we implemented the up/down arrow keys for navigation. @reidbarber might remember why but in the [ARIA pattern](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/), it's considered optional.

**Reply by AndresMangano** [reply]
Thanks for the answer and btw, what do you think are the biggest things missing regarding accessibility? @yihuiliao

**Reply by yihuiliao** [reply]
Everything that is required in the ARIA patterns has been implemented. Is there anything in particular that you have concerns about?
